cmd/deployIntegrationArtifact.go

```diff
@@ -75,7 +75,7 @@ func runDeployIntegrationArtifact(config *deployIntegrationArtifactOptions, tele
 
 	deployResp, httpErr := httpClient.SendRequest("POST", deployURL, nil, header, nil)
 
-	defer func() {
+	func() {
 		if deployResp != nil && deployResp.Body != nil {
```

Comment:
This if is in the CloseResponseBodyIfNecessary function. By having this dedicated function it is now possible to have one line for the defer, namely:
`defer cpi.CloseResponseBodyIfNecessary(deployResp)`

Reply:
It was decided to export this functionality in another PR